lab-sam/lib/greeting.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
```

Comment:
Great job!  Make sure to follow the instructions exactly, if it says a function that returns 'hello ' + name it should do exactly that.  Not relevant to this assignment though.